.github/workflows/ci.yml

```diff
@@ -53,23 +37,15 @@ jobs:
 
     - run: yarn --frozen-lockfile
     - run: yarn build-all
-    - run: yarn diff:sample-json
-    - run: yarn type-check
-    - run: yarn lint
-    - run: yarn test-proto # Run before unit-core because the roundtrip json is needed for proto tests.
 
     # Run tests that require headfull Chrome.
     - run: sudo apt-get install xvfb
```

Comment:
`build` stops being a great name for this job past this point. All the xvfb stuff could be moved to a new job. a couple of them require `dist` so it'd need to be dependent on `build` and download the artifact.

`smoke` will take so much longer than this job so it'd ndb, I prefer to keep it like this and not split the job.